src/api/ratelimits.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Rate Limits
+sidebar_order: 5
+---
+
+Sentry rate limits every API request made to prevent abuse and resource overuse. The limit is applied to each caller per endpoint.
+
+The rate limit follows a fixed window approach. Requests are counted into time buckets, determined by the window size. If the number of requests from a caller exceeds the number of allowed requests within a window, then that request will be rejected.
+
+While there is a default value, each endpoint can have their own maximum number of requests and window size. You can track your rate limit usage by looking at special headers in the response
```

Comment:
If defaults can be overwritten, should there be some mention on how to do it?

Reply:
Right now, rate limit overrides can only be done in the code. Is that something we want to include in this reference?

Reply:
No, let's not include it then.